worker/buildbot_worker/runprocess.py

```diff
@@ -810,15 +847,23 @@ def sendSig(self, interruptSignal):
 
     def _taskkill(self, pid, force):
         try:
-            if force:
-                cmd = "TASKKILL /F /PID {0} /T".format(pid)
-            else:
-                cmd = "TASKKILL /PID {0} /T".format(pid)
-
+            cmd = (
+                "TASKKILL /F /PID {0} /T".format(pid)
+                if force
+                else "TASKKILL /PID {0} /T".format(pid)
+            )
+            pr_info = win32job.QueryInformationJobObject(
+                self.job_object, win32job.JobObjectBasicProcessIdList
+            )
+            if runtime.platformType == 'win32' and hasattr(self, 'job_object'):
```

Comment:
QueryInformationJobObject and self.job_object are used couple of lines above without any checks.